gitbook-docs/market/user-guides/sign-up.md

```diff
@@ -1,24 +1,31 @@
 # Sign up
 
-Access to the market requires authentication. Please ensure you have a valid username/password. If you have not registered, please look at the instructions on how to do so [here](../front-end.md#new-user-signup) via the front end.
+Access to the market requires authentication.
+Make sure you have a valid username and password.
+If you have not registered, please look at the instructions on how to do so
+[via the web interface](../front-end.md#new-user-signup).
 
-Or, you can use the below command line in the `proof-market-toolchain` repository.
+Or, you can use the following command in the `proof-market-toolchain` repository.
 
 ## User
 
-Users can submit/retrieve orders on the proof market. You cannot generate proofs if you sign up as an ordinary user.
+Signup for an ordinary user through CLI looks like this:
 
-```
+```bash
 python3 scripts/signup.py user -u <username> -p <password> -e <e-mail>
 ```
 
-## Proof Producer
+Ordinary users can submit and retrieve orders on the Proof Market, but they can't generate proofs.
```

Comment:
Technically, users are the same for proof producing. It's just an additional action for the same account register as a proof producer

Reply:
Ok. "All users" then?

Reply:
Updated